kyverno-integration/modules/enforce/exceptions/backstage.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: kyverno.io/v2beta1
+kind: PolicyException
+metadata:
+  name: backstage-cnoe-operation
+  namespace: kyverno
+spec:
+  exceptions:
```

Comment:
Can we install backstage different to make it more align with best practice policies?

Reply:
Same as above